mage/algorithms/traditional-graph-analytics/degree-centrality.md

```diff
@@ -0,0 +1,48 @@
+---
+id: degree-centrality-algorithm
+title: Degree Centrality
+sidebar_label: Degree Centrality
+---
+
+## Description
+
+**Degree Centrality** is a method for estimating the centrality of graph 
+entities.
+It is a simple centrality measure that counts how many neighbors a node has. 
+For a directed graph, we have two versions of the measure: in-degree and 
+out-degree.
+In-degree counts the number of in-coming edges, and out-degree counts the number
+of out-going edges.
```

Comment:
Are in-coming and out-coming written like this on purpose or is it a mistake? Words are incoming and outgoing, but i don't know if this is some specific scenario

Reply:
Saw it written like that in a paper https://www.sci.unich.it/~francesc/teaching/network/degree.html
Thought it emphasizes the in and out of an edge, but I will change it to incoming and outgoing, it is understandable from that.